config/migrations/2024/20241113095422-clean-up-verkiezingen/20241113114927-purge-personen.sparql

```diff
@@ -0,0 +1,21 @@
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX person: <http://www.w3.org/ns/person#>
+
+DELETE {
+  GRAPH ?g {
+    ?persoon ?pp ?po .
+  }
+} WHERE {
+  GRAPH ?g {
+    ?persoon a person:Person .
+    ?res a mandaat:Verkiezingsresultaat .
+    ?res mandaat:isResultaatVan ?persoon .
+    FILTER NOT EXISTS {
+      ?res mandaat:isResultaatVoor ?lijst .
+    }
+    FILTER NOT EXISTS {
+      ?mandaat mandaat:isBestuurlijkeAliasVan ?persoon .
+    }
+    ?persoon ?pp ?po .
+  }
```

Comment:
pseudo: Remove person when there is a verkiezingsresultaat linked to it but no list or mandataris is found?

Aren't you than removing all persons that are currently not beiing used in the application?

Reply:
Well in an earlier migration I remove the specific kandidatenlijsten which we want removed. That is why I do the check
```
FILTER NOT EXISTS {
  ?res mandaat:isResultaatVoor ?lijst .
}
```
This ensure the person was on a kandidatenlijst, since he has a verkiezingsResultaat, but it is not linked to a kandidatenlijst anymore.
That way we only remove people that were elected, but just inserted in the wrong graph. The second check then ensures the person has no mandataris linked to it, because it could be that he was already elected a previous period.
Both checks together ensure, it are only people that were on a kandidatenlijst and are not used.